osmosis-1/osmosis.zone_assets.json

```diff
@@ -5570,6 +5572,35 @@
       ],
       "listing_date_time_utc": "2024-10-23T19:00:00Z",
       "_comment": "Mantra $OM"
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "ibc/21D8071EF5B02A86D945430D859A594CBF28287D38104A264BB9FD3B22BBF5DE",
+      "path": "transfer/channel-208/yum",
+      "osmosis_verified": false,
+      "osmosis_unlisted": true,
+      "listing_date_time_utc": "2024-10-25T01:00:00Z",
+      "canonical": {
+        "chain_name": "ethereum",
+        "base_denom": "0xcE682c89C63d2850Cb2ca898E44D6c7c30d897a6"
+      },
+      "_comment": "Project token for Cacao Swap",
+      "override_properties": {
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/yum.png"
+        }
+      },
+      "categories": [
+        "defi"
+      ],
+      "transfer_methods": [
+        {
+          "name": "Squid Bridge",
```

Comment:
Squid is already a build in provider for Osmosis Zone, so I don't think this will be necessary